### PR TITLE
Add information about creating a schedule to the readme

### PR DESCRIPTION
**Pull request by gianrubio**
Add schedule commands

### CODE REVIEW

**Comment by ncdc** [comment]
@gianrubio FYI markdown auto-numbers when it renders - that's why we use `1` repeatedly. If we end up inserting or removing steps, we don't have to renumber everything.

**Comment by gianrubio** [comment]
Didn't realise about that, rolling back now

**Comment by ncdc** [comment]
Also FYI we're reorganizing our docs - I'll defer to @Bradamant3 on this PR.

**Comment by gianrubio** [comment]
I was struggling on how to schedule backups that's why I added this section.

**Comment by ncdc** [comment]
Sure thing, can't hurt!

File: README.md [hunk 1]
```diff
@@ -101,6 +101,14 @@ Make sure that you install somewhere in your `$PATH`.
     
     NOTE: You might need to wait for a few minutes for the namespace to be fully cleaned up.
 
+### Schedule
+
+1. Run :
+
+    ```
+    ark schedule create nginx-backup --schedule="0 */1 * * *"
```
Comment (ncdc) [inline README.md]: Isn't `*/1` the same thing as `*`?

Comment (ncdc) [reply README.md]: We also support `@every 1h`

Comment (gianrubio) [reply README.md]: replaced to `@every 1h`

File: README.md [hunk 2]
```diff
@@ -101,6 +101,14 @@ Make sure that you install somewhere in your `$PATH`.
     
     NOTE: You might need to wait for a few minutes for the namespace to be fully cleaned up.
 
+### Schedule
```
Comment (ncdc) [inline README.md]: Maybe add some text saying that the step that created the backup was a one-time thing, and if you want to have Ark take backups at regularly scheduled intervals, you can do it like this. WDYT?

**Comment by Bradamant3** [comment]
If anybody comes back to give this PR some love: this content should now go into /docs/quickstart.md, not into the readme. We've taken the getting started content out of the readme and given it its own file.

**Comment by ncdc** [comment]
@gianrubio do you have availability to update the PR?

**Comment by nrb** [comment]
I've created #701 to carry this content over and address the comments by reviewers.

**Pull request closed without merging**